parameter_files/WebParameters_2dii.yml

```diff
@@ -1,6 +1,7 @@
 default:
     paths:
-        project_location_ext: ./
+        # Prefer "../PACTA_analysis/" -- not "./" nor "/path/to/PACTA_analysis"
+        project_location_ext: ../PACTA_analysis/
```

Comment:
The form "./" seems to work on docker but not locally. The form "../PACTA_analysis" works everywhere.